src/ansys/fluent/core/services/interceptors.py

```diff
@@ -133,13 +133,12 @@ def _intercept_call(
     ) -> Any:
         response = continuation(client_call_details, request)
         if response.exception() is not None and response.code() != grpc.StatusCode.OK:
-            ex = response.exception()
             new_ex_cls = RuntimeError
             try:
                 from google.rpc import error_details_pb2
                 from grpc_status import rpc_status
 
-                status = rpc_status.from_call(ex)
+                status = rpc_status.from_call(response)
```

Comment:
This fixes the ValueError which we are receving instead of RuntimeError for the issue scenatio.